docker/Dockerfile

```diff
@@ -73,51 +155,98 @@ RUN strip /usr/local/bin/grepperf \
    /usr/local/bin/getnetworkhistory \
    /usr/local/bin/isolatedServer \
    /usr/local/bin/getrewardhistory \
+   /usr/local/bin/buildTxBlockHashesToNums \
 #    /usr/local/bin/zilliqa \
    /usr/local/lib/libSchnorr.so \
    /usr/local/lib/libCryptoUtils.so \
    /usr/local/lib/libNAT.so \
    /usr/local/lib/libCommon.so \
    /usr/local/lib/libTrie.so
 
-FROM zilliqa/zilliqa-base:${ZILLIQA_BASE_VERSION}
+FROM zilliqa/scilla:${SCILLA_VERSION} AS scilla
+
+# run a copy -L to unfold the symlinkes, and strip all exes
+RUN mkdir -p /scilla/0/bin2/ && cp -L /scilla/0/bin/* /scilla/0/bin2/ && strip /scilla/0/bin2/*
+
+FROM ubuntu:20.04
```

Comment:
Easily forgotten when upgrading the base image. Consider an argument. See the next comment for a similar issue of repeating the list of packages.